### DISCUSSION TITLE
第3节课笔记(9班)

### DISCUSSION THREAD

**Discussion by RangeKing** [InternLM/Tutorial#67] [笔记]
请在下方提交第 3 节课笔记笔记链接，知乎/CSDN/飞书 等等都可。
笔记让知识记得更牢固💪

**Comment by xiaomile** [comment]
[基于 InternLM 和 LangChain 搭建你的知识库](https://blog.csdn.net/weixin_42567071/article/details/135475810)

**Comment by wangxuejiao** [comment]
[第3节课作业](https://zhuanlan.zhihu.com/p/677171567)

**Comment by ShelterWFF** [comment]
class3: http://t.csdnimg.cn/VcKcK

**Comment by KJ-Huang-sz** [comment]
https://blog.csdn.net/qq_40831032/article/details/135532147

**Comment by AIzealotwu** [comment]
https://blog.csdn.net/m0_72806612/article/details/135538524?spm=1001.2014.3001.5501

**Comment by liuxianyi** [comment]
[InternLM+langchain外挂知识库](https://liuxianyi.notion.site/InternLM-langChain-dcc7a0d113494090a573734f805583e7?pvs=4)

**Comment by haizeigh** [comment]
笔记 https://note.youdao.com/s/7ugn87qP
基于internlm和langchain搭建知识库视频 https://www.bilibili.com/video/BV1sT4y1p71V/?spm_id_from=333.788&vd_source=d6b5a09dda93b2c223da08c9b8241a1f

**Comment by xingyuan3** [comment]
笔记：https://www.mubu.com/doc/5Fuooi5UEzd

**Comment by SaigeRing** [comment]
https://zhuanlan.zhihu.com/p/677698804

**Comment by li4692625** [comment]
https://blog.csdn.net/li4692625/article/details/135565829

**Comment by nini0919** [comment]
第三节课笔记：https://www.notion.so/03-05ef623696954cb79844fc261e805ad9?pvs=4

**Comment by nxznm** [comment]
https://zhuanlan.zhihu.com/p/677351372

**Comment by wocclyl** [comment]
第三课笔记：https://m1ti1x9upsq.feishu.cn/wiki/EBqWwcGTsiqKnSkpHOBciDQhngf?from=from_copylink

**Comment by zhangzhidaSunny** [comment]
第三课笔记：
https://blog.csdn.net/m0_65719612/article/details/135581496
